pydov/types/interpretaties.py

```diff
@@ -722,3 +722,97 @@ class GeotechnischeCodering(AbstractBoringInterpretatie):
         'sourcefield': 'Y_mL72',
         'type': 'float'
     }]
+
+
+class QuartaireStratigrafieLaag(AbstractDovSubType):
+
+    _name = 'quartaire_stratigrafie_laag'
+    _rootpath = './/quartairstratigrafie/laag'
+
+    _fields = [{
+        'name': 'diepte_laag_van',
+        'source': 'xml',
+        'sourcefield': '/van',
+        'definition': 'diepte van de bovenkant van de laag '
+                      'quartairstratigrafie in meter',
+        'type': 'float',
+        'notnull': False
+    }, {
+        'name': 'diepte_laag_tot',
+        'source': 'xml',
+        'sourcefield': '/tot',
+        'definition': 'diepte van de onderkant van de laag '
+                      'quartairstratigrafie in meter',
+        'type': 'float',
+        'notnull': False
+    }, {
+        'name': 'lid1',
+        'source': 'xml',
+        'sourcefield': '/lid1',
+        'definition': 'eerste eenheid van de laag quartairstratigrafie',
+        'type': 'string',
+        'notnull': False
+    }, {
+        'name': 'relatie_lid1_lid2',
+        'source': 'xml',
+        'sourcefield': '/relatie_lid1_lid2',
+        'definition': 'verbinding of relatie tussen lid1 en lid2 van de '
+                      'laag quartairstratigrafie',
+        'type': 'string',
+        'notnull': False
+    }, {
+        'name': 'lid2',
+        'source': 'xml',
+        'sourcefield': '/lid2',
+        'definition': 'tweede eenheid van de laag quartairstratigrafie. '
+                      'Indien niet ingevuld wordt default dezelfde waarde '
+                      'als voor Lid1 ingevuld',
+        'type': 'string',
+        'notnull': False
+    }]
+
+
+class QuartaireStratigrafie(AbstractCommonInterpretatie):
```

Comment:
Quartairstratigrafie interpretaties are always linked to Boringen, so should be a subclass of AbstractBoringInterpretatie.

```suggestion
class QuartairStratigrafie(AbstractBoringInterpretatie):
```